K_deployment.md

```diff
@@ -1,410 +1,10 @@
 Once we have a working application, we're ready to deploy it. If you're not quite finished with your own application, don't worry. Just follow the [Up and Running Guide](http://www.phoenixframework.org/docs/up-and-running) to create a basic application to work with.
```

Comment:
This file diffed really poorly check it here http://www.phoenixframework.org/v0.9.0/docs/deployment